gym_xiangqi/piece.py

```diff
@@ -320,10 +352,18 @@ def get_actions(self, piece_id, state, actions):
         """
         Find legal moves for the soldiers
         """
-        if self.row > 4:
-            moves = [0]
+        if piece_id < 0:
+            low = RIVER_HIGH
+            high = BOARD_ROWS - 1
+            moves = [2]
```

Comment:
Could you add the comment explaining how the moves is indexing into the `ORTHOGONAL` field.

Reply:
How does it look now?

Reply:
Yeap LGTM.